### PR TITLE
US 1583733 Add missing language IDs - 46

### PR DESCRIPTION
**Pull request by v-thpra**
US 1583733 - Fix CATS P1 issues by adding missing language IDs to code blocks.
- Used "vb" for syntax examples of VB operators. 

Contributes to #2192

### CODE REVIEW

File: docs/visual-basic/language-reference/operators/addition-operator.md [hunk 1]
```diff
@@ -17,8 +17,12 @@ Adds two numbers or returns the positive value of a numeric expression. Can also
...
 ```vb
-expression1 + expression2  
-- or -  
+expression1 + expression2
+```
+  
+or
+
+```vb  
 + expression1
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/operators/addition-operator.md]:
```suggestion
+expression1  
```

File: docs/visual-basic/language-reference/operators/subtraction-operator.md [hunk 2]
```diff
@@ -23,9 +23,13 @@ Returns the difference between two numeric expressions or the negative value of
...
-      expression1 – expression2  
-- or -  
+```vb  
+expression1 – expression2
+```
+  
+or
+
+```vb  
 – expression1
```
Comment (rpetrusha) [inline docs/visual-basic/language-reference/operators/subtraction-operator.md]:
```suggestion
–expression1  
```

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for adding these language identifiers, @v-thpra. I've left two suggestions unrelated to your PR that I think make the syntax clearer.

**Comment by v-thpra** [comment]
@rpetrusha Thanks for the review. I implemented your suggestions.

**Comment by rpetrusha** [comment]
Thanks, @v-thpra. I'll merge now.

**Review by rpetrusha — APPROVED** [review]

**Pull request merged**